lib/tasks/portus.rake

```diff
@@ -49,4 +49,41 @@ namespace :portus do
       exit(-3)
     end
   end
+
+  desc "Update the manifest digest of tags"
+  task :update_tags, [:update] => [:environment] do |_, args|
+    # Warning
+    puts <<HERE
+This rake task may take a while depending on how many images have been stored
+in your private registry. If you are running this in production it's
+recommended that the registry is running in "readonly" mode, so there are no
+race conditions with concurrent accesses.
+
+HERE
+    print "Are you sure that you want to proceed with this ? (y/N) "
+    opt = $stdin.gets.strip
```

Comment:
Maybe it would be nice to have an environment variable that prevents this code from being invoked. That would allow conscious users to run the task in a non interactive way